packages/@webex/plugin-meetings/src/meeting/voicea-meeting.ts

```diff
@@ -91,31 +62,51 @@ export const processNewCaptions = ({data, meeting}) => {
       transcriptData,
       csisKey: key,
     });
+    const {speakerId} = speaker;
+    const interimId = `${transcriptId}_${speakerId}`;
 
     if (needsCaching) {
       transcriptData.speakerProxy[key] = speaker;
     }
-    const {speakerId} = speaker;
-    const interimId = `${transcriptId}_${speakerId}`;
+
     const captionData = {
       id: interimId,
       isFinal: data.isFinal,
       translations: value.translations,
       text: value.text,
-      currentCaptionLanguage: value.currentSpokenLanguage,
+      currentCaptionLanguage:
+        meeting.transcription?.languageOptions?.currentCaptionLanguage ||
```

Comment:
Can we explain these changes in the description a little more? I don't think these changes are covered there.
This is also applicable to the (isFinal) block that's been removed from earlier and added below. Can we explain that as well?

Perhaps it would also help to paste a transcription object for interim and final in the description for reference of the reviewer.

Reply:
Done. Please check